app/models/miq_schedule_worker/jobs.rb

```diff
@@ -12,8 +12,8 @@ def miq_server_status_update
   end
 
   def miq_server_worker_log_status
-    queue_work(:class_name  => "MiqServer", :method_name => "log_status",     :task_id => "log_status", :server_guid => MiqServer.my_guid, :priority => MiqQueue::HIGH_PRIORITY)
-    queue_work(:class_name  => "MiqWorker", :method_name => "log_status_all", :task_id => "log_status", :server_guid => MiqServer.my_guid, :priority => MiqQueue::HIGH_PRIORITY)
+    queue_work(:class_name  => "MiqServer", :method_name => "log_status", :tracking_label => "log_status", :server_guid => MiqServer.my_guid, :priority => MiqQueue::HIGH_PRIORITY)
+    queue_work(:class_name  => "MiqWorker", :method_name => "log_status_all", :tracking_label => "log_status", :server_guid => MiqServer.my_guid, :priority => MiqQueue::HIGH_PRIORITY)
```

Comment:
I think this to avoid logging multiple times, but we can probably dump it altogether.  Can't see why this would need a tracking label.